### ISSUE TITLE
aparte url voor Wet Open Overheid

### ISSUE THREAD

**Issue by gerardnienhuis** [IDgis/Onderzoeksbibliotheek#6]
er is nu al onderzoeksbank.dataportaaloverijssel.nl, daar moet bijkomen: woo.dataportaaloverijssel.nl

**Comment by HermanAssink** [comment]
Klopt het dat er zowel een extra productieomgeving als een extra acceptatieomgeving nodig is?

**Comment by gerardnienhuis** [comment]
ja, we hebben 1 ontwikkelomgeving voor wet open overheid en onderzoeksbank en "overige data" maar voor alle 3 deze systemen hebben we een aparte acceptatieomgeving en productieomgeving.
